online-shop/src/components/Product.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import TableRow from '@material-ui/core/TableRow';
+import TableBody from '@material-ui/core/TableBody';
+import Button from '@material-ui/core/Button';
+import {Route, Link, Switch} from 'react-router-dom';
+import ProductDetails from './ProductDetails';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import withStyles from '@material-ui/core/styles/withStyles';
+import createStyles from '@material-ui/core/styles/createStyles';
+import TableCell from '@material-ui/core/TableCell/TableCell';
+
+const MyCellType = withStyles((theme: Theme) =>
```

Comment:
Pune stilurile in fisiere de css (separate pentru fiecare componenta), pe cat posibil.